package.json

```diff
@@ -74,7 +74,7 @@
   },
   "license": "MIT",
   "engines": {
-    "node": ">=8.8.x"
+    "node": ">=12"
```

Comment:
This doesn't change the version of NodeJS used by CircleCI.

It still reports using `circleci/node:8.11-browsers`. You can update this here: https://github.com/angular/protractor/blob/master/.circleci/config.yml#L5

Reply:
This looks should be updated by another PR created be Keen https://github.com/angular/protractor/pull/5518/commits/76ee7ffd7699fc03726ead769cdc36c5853a2ffb

Reply:
Yeah, but that commit seems to not be in `master`:

> This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.

Reply:
Oh it's not merged yet: PR https://github.com/angular/protractor/pull/5518. I guess that we can wait on that PR to get merged before this one can be considered.